src/osd/ClassHandler.h

```diff
@@ -47,8 +46,7 @@ class ClassHandler
 
     void unregister();
 
-    ClassFilter() : fn(0)
-    {}
+    ClassFilter() : fn(0) {}
```

Comment:
i think `fn` is a function pointer. probably we can initialize it with `nullptr` and do this in-class?

Reply:
Fixed.